src/lib/sub-app/schema.json

```diff
@@ -7,11 +7,7 @@
     "name": {
       "type": "string",
       "description": "The name of the application.",
-      "$default": {
-        "$source": "argv",
-        "index": 0
-      },
-      "x-prompt": "What name would you like to use for the app?"
+      "default": "App"
```

Comment:
Why this has been changed?

Reply:
You right.
It was fixed in the last commit.

Reply:
It's still not fixed

Reply:
You right, I didn't push it. you can see it now :)